Makefile.am

```diff
@@ -187,6 +187,10 @@ motr_libmotr_la_LIBADD    = @MATH_LIBS@ @PTHREAD_LIBS@ @AIO_LIBS@ @RT_LIBS@ \
                             @YAML_LIBS@ @PROFILER_LIBS@ @UUID_LIBS@ @GALOIS_LIBS@ \
                             @DL_LIBS@ @CASSANDRA_LIBS@ @UV_LIBS@
 
+if ENABLE_LIBFAB
```

Comment:
Please rebase to libfabric_integration.

I think this chanhe is already merged.